tf_files/aws/modules/eks/cloud.tf

```diff
@@ -529,8 +536,24 @@ resource "aws_launch_configuration" "eks_launch_configuration" {
   }
 }
 
+# Create a new iam service linked role that we can grant access to KMS keys in other accounts
+# Needed if we need to bring up custom AMI's that have been encrypted using a kms key
+resource "aws_iam_service_linked_role" "autoscaling" {
+  aws_service_name = "autoscaling.amazonaws.com"
+  custom_suffix = "ASG"
```

Comment:
If there are multiple commons the arn will match for each and lead to tf errors. We should change this to something either random or specific to the common, so that there is not an overlap. Easiest solution is probably ${var.vpc_name}.

Reply:
You are 100% correct. I will update it!